### ISSUE TITLE
peercoind -conf argument not works

### ISSUE THREAD

**Issue by EminBudak** [peercoin/peercoin#413]
I run peercoind like that

`/usr/bin/peercoind -daemon -conf=/root/.peercoin/peercoin.conf > /dev/null 2>&1 &
`
But when I looked log file config file is not loaded.

Deamon starts sync without configuration.

I installed bionic-peercoind_0.6.4-1_amd64.deb

**Comment by EminBudak** [comment]
I think permissions are right. Because I run bitcoin, litecoin, dash and other clones properly like that. I faced up this problem with peercoin.

**Comment by backpacker69** [comment]
to further debug, please show output of these commands:

`ls -la /root/.peercoin/peercoin.conf
`
and while running peercoind daemon:

`ps auxw | grep peercoind
`

**Comment by backpacker69** [comment]
please start peercoind first and then run 'ps auxw | grep peercoind' command

**Comment by EminBudak** [comment]
I solved my problem thanks you change my viewpoint about problem. I focused the wrong point (conf file). I run deamon with bash script and it returns an error before executed because of port number 9901 already used. I changed my port number and it works now. The log file still doesn't consist configuration line but it loaded configuration. (Other bitcoin clones show conf line in log when it loaded I confused because of this.)
